.circleci/config.yml

```diff
@@ -388,7 +388,7 @@ commands:
             #  "Firefox is already running, but is not responding."
             # TODO: find out a way to shut down and restart firefox
       - upload-test-results
-  test-sockets-chrome:
+  test-sockets-chrome-command:
```

Comment:
Perhaps we should just inline this command instead of this change?  Maybe revert this part and we can do the inlinining as a separate chagne.